internal/api_server/server.go

```diff
@@ -126,6 +152,8 @@ func (s *Server) Run(ctx context.Context) error {
 	ws.RegisterRoutes(router)
 
 	srv := tlsmiddleware.NewHTTPServer(router, s.log, s.cfg.Service.Address)
+	srv.ReadTimeout = readTimeout
+	srv.MaxHeaderBytes = maxHeaderBytes
```

Comment:
Should these be moved into the server's constructor?

Reply:
Will do